frontend/src/views/WheresMyGeneV2/components/GeneSearchBar/components/ShareButton/utils.tsx

```diff
@@ -35,16 +39,17 @@ export const generateAndCopyShareUrl = ({
   if (organism && organism !== HUMAN_ORGANISM_ID) {
     url.searchParams.set("organism", organism);
   }
+
+  if (compare) {
```

Comment:
move this to be the first one to inject in the share link, so it continues to match the expected test share link order